### DISCUSSION TITLE
Azuracast docker update

### DISCUSSION THREAD

**Discussion by pawelzet** [AzuraCast/AzuraCast#4358] [Q&A]
Hello,

What is the last stable version of azuracast? Here I can see it is 0.12.3, I've update azuracast with:

`cd /var/azuracast
./docker.sh update-self
./docker.sh update`

And still my version is 12.2. On your changelog there is a version of 0.13.0 with podcasts. Is it stable now?

Thank you in advance for your help.

**Comment by ItsMitchh** [comment]
hello, our latest update is 0.13.0 and its the latest stable version, you can update :)

**Comment by pawelzet** [comment]
> hello, our latest update is 0.13.0 and its the latest stable version, you can update :)

I've tried to, it was said that azuracast was succesfully update, in the panel I have still 0.12.2 version. Any ideas why it is like that?

Paweł

**Comment by ItsMitchh** [comment]
You can try and switch the `release channel` to see if it resolves this issue, what I suggest is switching to Rolling THEN back to Stable, and attempting the update. 

https://docs.azuracast.com/en/getting-started/updates/release-channels

**Comment by Gstoica01** [comment]
i have an error, to switch to stable version
`
New Docker utility script downloaded.
You can now re-run any previous command with the updated utility script.
`
when i run the command
`./docker.sh update-self`

**Reply by LordHelmchen666** [reply]
That is not an error. After this you can run  `./docker.sh update`

**Comment by Gstoica01** [comment]
can't change to stable version
`root@azuracast:/var/azuracast/www# ./docker.sh setup-release
Your current release channel is 'Rolling Release'. Switch to 'Stable' release channel? [y/N] y
You should update the Docker Utility Script after changing release channels. Automatically update it now? [Y/n] Y
New Docker utility script downloaded.
You can now re-run any previous command with the updated utility script.
root@azuracast:/var/azuracast/www# `

**Reply by DJMic** [reply]
You need to run `./docker.sh update` as mentionned by LordHelmchen666.
<img width="733" height="304" alt="image" src="https://github.com/user-attachments/assets/9498489b-feef-460d-98f6-21d111192aae" />

**Reply by LordHelmchen666** [reply]
`root@azuracast:/var/azuracast/www`
I guess you are inside the docker container. Type `exit`, then you should be in /var/azuracast. There you can run commands again.

**Comment by Gstoica01** [comment]
ok
but not switch to stable release
`
root@azuracast:/var/azuracast/www# ./docker.sh update-self
New Docker utility script downloaded.
You can now re-run any previous command with the updated utility script.
root@azuracast:/var/azuracast/www# ./docker.sh update
[NOTICE] Before you continue, please make sure you have a recent snapshot of your system and or backed it up.
Are you ready to continue with the update? [Y/n] Y
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
./docker.sh: line 607: [: : integer expression expected
Docker is out of date on this server. Attempt automatic upgrade? [Y/n] Y
# Executing docker install script, commit: 1ebc10ccd106153669a1e7b5984497e0fcf9d3b1
Warning: the "docker" command appears to already exist on this system.

If you already have Docker installed, this script can cause trouble, which is
why we're displaying this warning and provide the opportunity to cancel the
installation.

If you installed the current Docker package using this script and are using it
again to update Docker, you can ignore this message, but be aware that the
script resets any custom changes in the deb and rpm repo configuration
files to match the parameters passed to the script.

You may press Ctrl+C now to abort this script.
+ sleep 20
+ sh -c apt-get -qq update >/dev/null
+ sh -c DEBIAN_FRONTEND=noninteractive apt-get -y -qq install ca-certificates curl >/dev/null
+ sh -c install -m 0755 -d /etc/apt/keyrings
+ sh -c curl -fsSL "https://download.docker.com/linux/debian/gpg" -o /etc/apt/keyrings/docker.asc
+ sh -c chmod a+r /etc/apt/keyrings/docker.asc
+ sh -c echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/debian bookworm stable" > /etc/apt/sources.list.d/docker.list
+ sh -c apt-get -qq update >/dev/null
+ sh -c DEBIAN_FRONTEND=noninteractive apt-get -y -qq install docker-ce docker-ce-cli containerd.io docker-compose-plugin docker-ce-rootless-extras docker-buildx-plugin docker-model-plugin >/dev/null

================================================================================

To run Docker as a non-privileged user, consider setting up the
Docker daemon in rootless mode for your user:

    dockerd-rootless-setuptool.sh install

Visit https://docs.docker.com/go/rootless/ to learn about rootless mode.


To run the Docker daemon as a fully privileged service, but granting non-root
users access, refer to https://docs.docker.com/go/daemon-access/

WARNING: Access to the remote API on a privileged Docker daemon is equivalent
         to root access on the host. Refer to the 'Docker daemon attack surface'
         documentation for details: https://docs.docker.com/go/attack-surface/

================================================================================

Installing Docker Compose...
Docker Compose updated!
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Clean up all stopped Docker containers and images to save space? [Y/n] Y
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Update complete!
root@azuracast:/var/azuracast/www# ./docker.sh update-self
New Docker utility script downloaded.
You can now re-run any previous command with the updated utility script.
root@azuracast:/var/azuracast/www# ./docker.sh setup-release
Your current release channel is 'Stable'. Switch to 'Rolling Release' release channel? [y/N] N
root@azuracast:/var/azuracast/www# `

it remains to rolling release
`Powered by [AzuraCast](https://www.azuracast.com/) • Rolling Release #[9383217](https://github.com/AzuraCast/AzuraCast/commit/93832174ac9cc1fbab85020f1954f6c09155c41a) (2025-08-29 6:44) • Docker • PHP 8.4
Like our software? [Donate to support AzuraCast!](https://donate.azuracast.com/)`

**Reply by LordHelmchen666** [reply]
`root@azuracast:/var/azuracast/www`
It seems you are still INSIDE your docker container. How did you install Azuracast?

**Comment by Gstoica01** [comment]
Is instaled on docker compose (portainer)

**Reply by LordHelmchen666** [reply]
What's wrong with you? You are getting answer here https://github.com/AzuraCast/AzuraCast/issues/8099#issuecomment-3255402947

**Comment by Gstoica01** [comment]
nothing in this topic or the other one changes the version of azuracast

**Reply by LordHelmchen666** [reply]
That topic describes exactly how to do it.

**Comment by Gstoica01** [comment]
I followed step by step, but the channel doesn't change.
